app/retail/templates/settings/tax.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'settings/settings.html' %}
+{% load i18n static %}
+{% block settings_content %}
+<form id="settings" name="location" method="POST">
+  <div class="form-group">
+    {% include 'ftux/tax.html' %}
+  </div>
+  {% csrf_token %}
+  <div class="col-md-8 m-auto">
+    <button class='button button--primary' type="button" onClick="save_tax_settings()">{% trans "Save" %}</button>
+  </div>
+</form>
+{% endblock %}
+{% block scripts %}
+<script>
+  var csrftoken = jQuery("[name=csrfmiddlewaretoken]").val();
+</script>
+<script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBaJ6gEXMqjw0Y7d5Ps9VvelzOOvfV6BvQ&libraries=places&callback=initPlacecomplete"></script>
```

Comment:
might make sense to move the API KEY somewhere else to DRY it??

Reply:
Hi, i just copied the approach used in `job.html`.  how would you like to do it ?

Reply:
I moved the api key on backend view. do you agree or you mean something different. sorry if i did not understand it.